spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -9,25 +9,27 @@
 
   let(:stripe_event) do
     double(Stripe::Event, type: "checkout.session.completed",
-      data: double(object: double(payment_intent: "pi_1234",
-        customer_details: double(email: "test@example.com"),
-        shipping: double(name: "Test",
-          address: double(line1: "123 N West", line2: "apt 1", city: "Oakland", state: "CA", postal_code: "94609")))))
+      data: double(object: double(payment_intent: "pi_1234", customer_details: double(email: "test@example.com"))))
   end
 
   let(:balance_transaction) { double(Stripe::BalanceTransaction, fee: 100) }
 
   let(:payment_intent) do
     double(Stripe::PaymentIntent, transfer_group: order.id,
-      charges: [double(Stripe::Charge, balance_transaction: "btx_2234")])
+      latest_charge: "ch_1234")
```

Comment:
This probably all fits on the previous line.

Reply:
It do!